flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/agg/AggsHandlerCodeGenerator.scala

```diff
@@ -700,14 +841,21 @@ class AggsHandlerCodeGenerator(
     """.stripMargin
   }
 
+  private def genEmitValue(): String = {
```

Comment:
this method can be internal in `generateTableAggsHandler`?

Reply:
This method would also be used for the window flatAggregate, i.e., would also be used in generateWindowTableAggsHandler() later.